rpmlint/checks/SourceCheck.py

```diff
@@ -59,7 +60,7 @@ def _check_permissions(self, fname, pkgfile, pkg):
 
     def _check_compressed_source(self, fname, pkg):
         """
-        Check if the Source is compressed if CompressExtension configuration options is used (xz, gz, tgz or bz2).
+        Check if the Source is compressed if CompressExtension configuration options is used (xz, gz, tgz or bz2, zst).
```

Comment:
it looks weird now, could you just remove the or? even better sort it alphabetically and keep the or prior xz :)

rest of the pr looks great.

Reply:
it should be `(gz, tgz, bz2, xz, or zstd)`

Reply:
@scarabeusiv  Thanks for catching this. Fixed as @Conan-Kudo requested.